content/glossary/artifact_tree.md

```diff
@@ -101,3 +103,65 @@ flowchart BT
     Artifact-6[Artifact-6 ID] --> Artifact-3[Artifact-3 ID]
     Artifact-7[Artifact-7 ID] --> Artifact-3[Artifact-3 ID]
 ```
+
+GitBOM advocates for using the [Git Ref](/glossary/git/#git-ref) of an artifact as its [artifact id](/glossary/artifact#artifact-identifiers):
```

Comment:
I think the recommendation is a bit more specific than just a cryptographic hash.

Reply:
Let's keep it as is as it seems to be well understood by many even though the term git ref is overloaded and not well defined. There's still a set of questions from those not familiar with git that I hope to avoid. I will suggest a change lower in the file in hopes of resolving this concern.